source/isaaclab/isaaclab/assets/articulation/articulation_data.py

```diff
@@ -509,6 +510,22 @@ def body_acc_w(self):
             self._body_acc_w.timestamp = self._sim_timestamp
         return self._body_acc_w.data
 
+    @property
+    def body_incoming_joint_wrench_b(self) -> torch.Tensor:
+        """Joint reaction wrench applied from body parent to child body in parent body frame.
+
+        Shape is (num_instances, num_bodies, 6). All body reaction wrenches are provided including the root body to the
+        world of an articulation.
+
+        `Physx documentation` https://nvidia-omniverse.github.io/PhysX/physx/5.5.1/docs/Articulations.html#link-incoming-joint-force
+        `Physics tensors API`: https://docs.omniverse.nvidia.com/kit/docs/omni_physics/latest/extensions/runtime/source/omni.physics.tensors/docs/api/python.html#omni.physics.tensors.impl.api.ArticulationView.get_link_incoming_joint_force
```

Comment:
Better to use html links properly :)

```suggestion
        For more information on joint wrenches, please check the`PhysX documentation <https://nvidia-omniverse.github.io/PhysX/physx/5.5.1/docs/Articulations.html#link-incoming-joint-force>`__
        and the underlying `PhysX Tensor API <https://docs.omniverse.nvidia.com/kit/docs/omni_physics/latest/extensions/runtime/source/omni.physics.tensors/docs/api/python.html#omni.physics.tensors.impl.api.ArticulationView.get_link_incoming_joint_force>`__ .
```